staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto

```diff
@@ -177,11 +187,17 @@ message Condition {
   // The value should be a CamelCase string.
   // This field may not be empty.
   // +required
+  // +kubebuilder:validation:Required
+  // +kubebuilder:validation:MaxLength=1024
+  // +kubebuilder:validation:MinLength=1
+  // +kubebuilder:validation:Pattern=`^[A-Za-z]([A-Za-z0-9_,:]*[A-Za-z0-9_])?$`
```

Comment:
Allowing commas.  Not a fan....

Reply:
Same for colon.  Hrm... gonna have to think about this.

Reply:
> Allowing commas. Not a fan....

Matches the reality of the world I'm afraid.  We have people interested in adopting now that have union reasons with delimiters

Reply:
I guess since we said "machine readable" this is the world we live in.